File: Java/loops3.java
// Print first N odd numbers
import java.util.Scanner;

public class loops3 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter N: ");
        int n = sc.nextInt();

        for (int i = 1; i <= n; i++) {
            System.out.print((2 * i - 1) + " ");
        }
        System.out.println();
    }
}

File: Java_Practice/CPP/loops3.cpp
//first N odd numbers
#include<iostream>
using namespace std;
int main()
{
    int n;
    cout<<"ENTER N : "<<endl;
    cin>>n;
    for(int i = 1 ; i<=n ; i++){
        cout<<"first N odd numbers : "<<2*i-1<<endl;
    }
    return 0;
}